docs/sdk/1-workshop/03-attester/02-did.md

```diff
@@ -25,7 +25,7 @@ Take a look at our [DID documentation](/docs/sdk/core-feature/did) if you want t
 
 There are currently four different key types that a DID supports:
 
-- An _authentication keypair_, used to sign claims and presented them when requested
+- An _authentication keypair_, used to sign claims and credential presentations
```

Comment:
```suggestion
- An _authentication keypair_, used to sign claims and create authenticated credential presentations
```

Reply:
It would make sense to note that you have to sign a presentation but would be nice to keep this description brief